packages/mask/src/plugins/Savings/constants.ts

```diff
@@ -1,2 +1,235 @@
+import { ChainId, createERC20Tokens, createNativeToken, FungibleTokenDetailed } from '@masknet/web3-shared-evm'
+
 export const SAVINGS_PLUGIN_NAME = 'Savings'
 export const SAVINGS_PLUGIN_ID = 'com.savings'
+
+export const LDO_PAIRS: [FungibleTokenDetailed, FungibleTokenDetailed][] = [
+    [
+        createNativeToken(ChainId.Mainnet),
+        createERC20Tokens('LDO_ADDRESS', 'Lido DAO Token', 'LDO', 18)[ChainId.Mainnet],
+    ],
+]
+
+export const AAVE_PAIRS: [FungibleTokenDetailed, FungibleTokenDetailed][] = [
+    [
+        createERC20Tokens('USDT_ADDRESS', 'Tether USD', 'USDT', 6)[ChainId.Mainnet],
+        createERC20Tokens('aUSDT_ADDRESS', 'Aave Interest bearing USDT', 'aUSDT', 6)[ChainId.Mainnet],
+    ],
+    // {
```

Comment:
Hi @layinka Plz help to update AAVE pairs.

Reply:
Done